guides/client/js-interop.md

```diff
@@ -354,3 +354,47 @@ Hooks.Chart = {
 ```
 
 *Note*: In case a LiveView pushes events and renders content, `handleEvent` callbacks are invoked after the page is updated. Therefore, if the LiveView redirects at the same time it pushes events, callbacks won't be invoked on the old page's elements. Callbacks would be invoked on the redirected page's newly mounted hook elements.
+
+## Locked DOM elements and loading states
```

Comment:
It's good to have this in the docs. I feel like we're going to need to add some examples where this is necessary for good UX.